src/Homie/Helpers.cpp

```diff
@@ -18,13 +18,13 @@ const char* Helpers::getDeviceId() {
 MdnsQueryResult Helpers::mdnsQuery(const char* service) {
   MdnsQueryResult result;
   result.success = false;
-  int n = MDNS.queryService(service, "tcp");
+  int n = 0; // MDNS.queryService(service, "tcp");
```

Comment:
It was included in the pull request, because I just committed my local version based on v1.5.0.

I did not had the time to create a "clean" version, but wanted to show the implementation of the virtual methods, before I started a 5 day business trip.

As your pull request implements the same solution, I will close my pull request.